tests/measureSources.py

```diff
@@ -294,6 +302,31 @@ def testPixelFlags(self):
             source,
             exp,
         )
+        # Test that if there is no center and centroider that the object should look at the footprint
+        plugin, cat = makePluginAndCat(measBase.PixelFlagsAlgorithm, "test", control)
```

Comment:
In fact there are three tests in quick succession here, and the first one tests what happens if there's no `Footprint`! 
Since you're following a very similar pattern repeatedly (make record, attempt to measure, check for failure), I wonder if this should actually be a series of separate tests calling a common implementation?

Also: a few lines above there's a comment (which GitHub won't let me annotate, because it's not been changed) to the effect that "the new code in meas_base InputUtilities.py now throws...". Is this code still testing `InputUtilities`, or is it testing your new code above? Please clarify the comment.